src/lmstudio/json_api.py

```diff
@@ -418,10 +418,32 @@ def __init__(self, message: str) -> None:
         super().__init__(message, None)
 
 
+@sdk_public_type
+
 @sdk_public_type
 class LMStudioPredictionError(LMStudioServerError):
     """Problems reported by the LM Studio instance during a model prediction."""
 
+@sdk_public_type
```

Comment:
The more granular subclasses and the session scoped fixture loading would be easier to follow if placed in different PRs.